cartographer/mapping_3d/scan_matching/ceres_scan_matcher_test.cc

```diff
@@ -34,10 +34,9 @@ namespace {
 class CeresScanMatcherTest : public ::testing::Test {
  protected:
   CeresScanMatcherTest()
-      : hybrid_grid_(1.f /* resolution */,
-                     Eigen::Vector3f(0.5f, 0.5f, 0.5f) /* origin */),
+      : hybrid_grid_(1.f /* resolution */),
```

Comment:
remove comment? We are inconsistent in having it or not and I feel it is not necessary, it seems understandable without.